src/with-font-loading.js

```diff
@@ -48,18 +49,23 @@ const withFontLoading = (fontName: string) => {
       }
 
       componentDidMount() {
+        this.mounted = true;
         if (this.state.$fontStyles.fontFamily !== fontName) {
-          // $FlowFixMe
+          // $FlowFixMe (this can only be browser so will not be undefined)
```

Comment:
If you did want to get rid of this, Flow's [type refinement](https://flow.org/en/docs/lang/refinements/) may help here:

```
if(loadFont) {
   loadFont(...);
}
```

Either seems reasonable to me though, given this is a limitation of Flow + isomorphic code.

Reply:
I don't think that would work because the function is 

```js
export default function(font: string) {
  if (__BROWSER__ && document) {
    // $FlowFixMe
    return document.fonts && typeof document.fonts.load === 'function'
      ? document.fonts.load(`1em ${font}`) // native API requires size
      : loadFontPolyfill(font);
  }
}
```

i.e. the function is always there but if NODE it returns undefined